.github/workflows/image.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           context: .
           platforms: linux/amd64
           push: true
-          tags: quay.io/sustainable_computing_io/kepler
+          tags: quay.io/sustainable_computing_io/kepler:latest
```

Comment:
per [instruction](https://github.com/marketplace/actions/build-and-push-docker-images), `tags` is what we need to set the image tag